releases/release-1.22/release-team.md

```diff
@@ -1,16 +0,0 @@
-# Kubernetes 1.22 Release Team
```

Comment:
same here, I don't think this file should be deleted?

Reply:
Good catch. Looking into it

Reply:
Resolved it - https://github.com/savitharaghunathan/sig-release/tree/bootstrap-release-1.22/releases/release-1.22